src/dev/ci_setup/setup.sh

```diff
@@ -41,13 +41,47 @@ fi
 
 
 ###
-### "install" node by extending the path with it's bin directory
+### "install" node into this shell
 ###
 export PATH="$nodeDir/bin:$PATH"
+hash -r
+
+###
+### downloading yarn
+###
+yarnVersion="$(node -e "console.log(String(require('./package.json').engines.yarn || '').replace(/^[^\d]+/,''))")"
```

Comment:
Shouldn't we be using `tasks/vendor/yarn-1.3.2-with-ignore-fix.js` or am I missing something?

Reply:
Yeah, we should probably do that. I'll fix.

Reply:
We're going to need to put this back once https://github.com/yarnpkg/yarn/pull/5059 is merged but 🤷‍♂️